RealmModelGenerator/ViewController.swift

```diff
@@ -8,32 +8,29 @@
 
 import Cocoa
 
-class ViewController: NSViewController, NSTableViewDelegate, NSTableViewDataSource {
+class ViewController: NSViewController {
     static let TAG = NSStringFromClass(ViewController)
     
     var schema: Schema = Schema()
     
-    //    //MARK: - NSTableViewDataSource
-    //    func numberOfRowsInTableView(tableView: NSTableView) -> Int {
-    //        return 1000
-    //    }
-    //
-    //    //MARK: - NSTableViewDelegate
-    //    func tableView(tableView: NSTableView, viewForTableColumn tableColumn: NSTableColumn?, row: Int) -> NSView? {
-    //        let cell = tableView.makeViewWithIdentifier(TitleCell.IDENTIFIER, owner: nil) as! TitleCell
-    //        cell.title = "\(row)"
-    //        return cell
-    //    }
+    @IBOutlet weak var leftDivider: NSView!
+    @IBOutlet weak var rightDivider: NSView!
+    @IBOutlet weak var leftContainer: NSView!
```

Comment:
we shouldn't really need to have references to this views in the code. Only really references to the view controllers.
If we can make the view controllers IBoutlets that would be amazing